addons/mail/static/src/messaging/component/composer/composer.xml

```diff
@@ -78,7 +78,7 @@
                             <div class="o_Composer_primaryToolButtons">
                                 <Popover position="'top'" t-on-o-emoji-selection="_onEmojiSelection" t-ref="emojisPopover">
                                     <!-- TODO FIXME o-open not possible to code due to https://github.com/odoo/owl/issues/693 -->
-                                    <button class="o_Composer_button o_Composer_buttonEmojis o_Composer_toolButton btn btn-secondary"
+                                    <button class="o_Composer_button o_Composer_buttonEmojis o_Composer_toolButton btn"
```

Comment:
I think we should keep `btn-secondary` but explicitly set the color on those buttons, in order to not inherit green color. Attachment button seems to not have this issue because a more specific CSS rule set gray color.